zmon-agent.py

```diff
@@ -261,7 +266,12 @@ def main():
 
     if not args.region:
         logging.info("Trying to figure out region...")
-        region = boto.utils.get_instance_metadata()['placement']['availability-zone'][:-1]
+        try:
+            response = requests.get('http://169.254.169.254/latest/meta-data/placement/availability-zone', timeout=2)
```

Comment:
this looks wrong, you still need to cut off the last char to get the region (now you have the AZ)